app/routes/reductions.js

```diff
@@ -133,17 +134,65 @@ module.exports = function (router) {
     })
   })
 
+  router.post('/:organisationLevel/:id/update-reduction-status', function (req, res, next) {
+    var organisationLevel = req.params.organisationLevel
+
+    if (organisationLevel !== organisationUnitConstants.OFFENDER_MANAGER.name) {
+      throw new Error('Only available for offender manager')
+    }
+
+    var reductionStatus = req.body.status
+    var id = req.params.id
+    var reductionId = req.body.reductionId
+
+    if (!requestStatusVerified(reductionStatus)) {
+      return res.redirect(302, '/' + organisationLevel + '/' + id + '/reductions?fail=true')
+    }
+
+    return reductionsService.updateReductionStatus(id, reductionId, reductionStatus)
```

Comment:
Need to catch the error here, can see the ones Tom has added to overview route as example

Reply:
I have added it for all routes in the reduction.js file.